README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please try to use the template from Microverse as a ReadMe

Reply:
- [OPTIONAL] Nice one @wassimchakib, but try as much as you can to include `WHY` the student is asked to make of a suggestion as this would make it clearer for the student why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where where more information can got for  more clarification.